components/consumers/pdf/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: container publish
+
+CONTAINER_REPO=
+SMITHY_VERSION=
+SOURCE_CODE_REPO=
+PRODUCER_AGGREGATOR_BASE_IMAGE=$(shell test -e .custom_image && cat .custom_image || echo "scratch")
+
+DOCKER=docker
+
+container:
+	$(eval workdir:=$(shell mktemp -d /tmp/pdf.XXXXXX))
```

Comment:
Can we make sure that we're using the correct `--platform` to build the container?

There are a few examples in the repo between Makefile and scripts :)

It would help me running this locally